refgenie/asset_build_packages.py

```diff
@@ -11,11 +11,6 @@
 #   provided via the CLI. These should be listed as 'required_inputs' and
 #   will be checked for existence before the commands are executed.
 
-
-TSS_AWK = "'{if($4==\"+\"){print $3\"\t\"$5\"\t\"$5\"\t\"$4\"\t\"$13}else{print $3\"\t\"$6\"\t\"$6\"\t\"$4\"\t\"$13}}'"
-GENE_ANNO_FILE_NAME = "refGene.txt"
-GENE_ANNO_NAME = "gene_anno"
-
 asset_build_packages = {
     "fasta": {
         "description": "Given a gzipped fasta file, produces fasta, fai, and chrom_sizes assets",
```

Comment:
should we change the `description` keys to `asset_description` so that they match the standard? 

https://github.com/databio/refgenconf/blob/ad4cc28fc2204a022d3ff92839be9494630e7293/refgenconf/const.py#L25

Reply:
Also, I feel like the descriptions should rather describe the result of the recipe -- asset. This is the displayed on http://refgenomes.databio.org

Potentially, we could treat `description` as the description of the recipe (as it is now, I suppose) and _new_ `asset_description` key as the description of the asset itself.

Reply:
I like your second thought here, the idea of a "recipe description" (i.e. the current description), and an "asset description" which describes the outcome (i.e. the asset itself)

Reply:
yes. some asset packages have more than one asset. we would need an `asset_description` for each asset. So, the asset package format will need to change slightly to accommodate this.

Reply:
(that's not a problem since the only thing using this is refgenie at the moment)

Reply:
I've made the changes to the recipes format. Wasn't sure how to precisely describe some of the assets, so I left the values empty and tagged them with `#TODO`